tck/features/expressions/precedence/Precedence1.feature

```diff
@@ -476,4 +517,4 @@ Feature: Precedence1 - On boolean values
       | boolop |
       | OR     |
       | XOR    |
-      | AND    |
+      | AND    |
```

Comment:
Nitpicking, but there should actually be a newline terminating the file, right?

Reply:
Fixed.